platform/state/platform_state.proto

```diff
@@ -195,3 +211,134 @@ message MinimumJudgeInfo {
      */
     uint64 minimum_judge_ancient_threshold = 2;
 }
+
+/**
+ * A network address book.<br/>
+ * The address book records the address of every known consensus node that
+ * participates in the network, including `0 weight` nodes.<br/>
+ */
+message AddressBook {
+  /**
+   * A consensus round.<br/>
+   * The round when this address book was created.
+   */
+  int64 round = 1;
+
+  /**
+   * The node ID of the next address that can be added.
+   * <p>
+   * The next node identifier assigned SHALL be equal to this value.<br/>
+   * All existing node identifiers SHALL be strictly less than this value.<br/>
+   */
+  NodeId next_node_id = 2;
+
+  /**
+   * A list of all consensus node addresses.
+   * <p>
+   * This list SHALL NOT be empty.
+   * If a consensus node is not in this list it SHALL NOT participate in the
+   * network.
+   */
+  repeated Address addresses = 3;
+}
+
+/**
+ * A single network address.
+ * This is one address in the network address book, including all required
+ * information to include that consensus node in the consensus gossip.
+ *
+ * ### Implementation Detail
+ * When `enableEventStreaming` is set to true, the memo field is REQUIRED
+ * and MUST be unique.
```

Comment:
What is memo used for that this requirement exists?
Also, could we just rename the field in this temporary structure to describe how we are _actually_ using it?

Reply:
This comment comes from `com.swirlds.platform.system.address.AddressBook` javadoc. I took it at face value, but at the second glance I couldn't find any justification for this requirement, so I remove this comment